### ISSUE TITLE
In SN post, show links from other SN posts to it

### ISSUE THREAD

**Issue by kristapsk** [stackernews/stacker.news#667]
Back in the day, some 15-20 years ago, in blogging there was that [trackback](https://en.wikipedia.org/wiki/Trackback) thing when some blog post contained link to some other blog post it notified other site about that and it could be shown like a comment below the post. Ok, that thing is probably dead today. But it would be useful internally for SN to show references to SN post from other SN posts. For example, I mentioned [Heroes of Might and Magic III](https://stacker.news/items/340167) post in my post about [King's Bounty](https://stacker.news/items/341376). Would be cool that in HoMMIII post visitor could see that it is mentioned in KB post.

**Comment by huumn** [comment]
Notify the author: https://stacker.news/items/438451

**Comment by abhiShandy** [comment]
Would you like to add a cost to cite other posts since it is going to create a comment (and comments are not free)?

**Comment by ekzyis** [comment]
No, no cost, we want to incentivize linking to other content for evergreenness

I also don't think we want to have it as replies, maybe references could be in the "related" section with some special indicator or inside an own section? But these sections don't show for comments (even if they are the root item of the page), so mh.

**Comment by huumn** [comment]
Agreed, it shouldn't create a comment. 

Without a better idea, I think it should just be a link in the overflow menu, ie the `...` on the item perhaps shown as `X references`.

When clicked it takes you to a page like the related page: https://stacker.news/items/522161/related

**Comment by ekzyis** [comment]
We could also show it in the item info `100 sats \ 3 comments \ 5 references` if references > 0. That would work for comments, too.

But maybe that's too prominent

**Comment by huumn** [comment]
It's mostly the screen real estate I'm concerned about.

Maybe: `100 sats \ 3 comments \ 5 refs`

Still, a lot to look at. Alternatively:

<img width="918" alt="Screenshot 2024-04-28 at 5 06 58 PM" src="https://github.com/stackernews/stacker.news/assets/34140557/4bc868d6-0745-438b-acea-48c9f32ab156">

**Comment by dillon-co** [comment]
what about a second dropdown next to or under `related posts` labeled `referenced by` ?
